sdk/core/azure-core/src/main/resources/META-INF/native-image/com.azure/azure-core/native-image.properties

```diff
@@ -11,6 +11,7 @@ Args=\
                              ch.qos.logback.core.status.StatusBase,\
                              ch.qos.logback.core.util.Loader,\
                              ch.qos.logback.core.util.StatusPrinter,\
+                             ch.qos.logback.core.util.StatusPrinter2,\
```

Comment:
Is there a system we could put into place which checks all these?

Reply:
Yeah, we need to scan for all new types in the libraries we use and add them to this list. Unfortunately, GraalVM doesn't support adding just the package names with wildcards to include everything in the package - https://github.com/Azure/azure-sdk-for-java/issues/41519